src/group_theory/perm/basic.lean

```diff
@@ -287,6 +287,18 @@ equiv.ext $ λ ⟨x, hx⟩, by { dsimp [subtype_perm, of_subtype],
 
 @[simp] lemma default_perm {n : Type*} : default (equiv.perm n) = 1 := rfl
 
+/-- Permutations on a subtype are equivalent to permutations on the original type that fix pointwise
+the rest. -/
+@[simps] protected def subtype_equiv (p : α → Prop) [decidable_pred p] :
```

Comment:
I don't think this is a good name - this is far too similar in name to `equiv.subtype_equiv` yet has a completely different statement

Reply:
Perhaps `equiv.perm.subtype_equiv_subtype_perm` or `equiv.perm_subtype_equiv_subtype_perm`?

Reply:
Arf. But it perfectly describes what it does. Could maybe `equiv.subtype_equiv_subtype` get renamed to `equiv.subtype_equiv_subtype`, `equiv.subtype_ext` or `equiv.subtype_ext_equiv`?

Reply:
If you rename `equiv.subtype_equiv` then you should rename `equiv.perm.subtype_perm` too, as they're the same.

But I dispute the claim that it perfectly describes what it does - all you've done is describe the LHS; `perm_subtype_equiv_subtype_perm` is a more complete description

Reply:
And what about namespaces? Should it be `equiv.perm.subtype_equiv_subtype_perm`? Or `equiv.perm_subtype_equiv_subtype_perm `? Or plainly `perm_subtype_equiv_subtype_perm`?